<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Retinotopic_Mapping_DialogClass</class>
 <widget class="QDialog" name="Retinotopic_Mapping_DialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Retinotopic_Mapping_Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>650</y>
     <width>221</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="mainTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>357</width>
     <height>220</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>220</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="experimentTab">
    <attribute name="title">
     <string>Experiment</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldsStayAtSizeHint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="experimentsFolderLbl">
         <property name="text">
          <string>Experiments folder:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="experimentsFolderEd">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="BrowseExpFolderBtn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="experimentsLbl">
         <property name="text">
          <string>Experiments:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="ExperimentCmb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="ExperimentTypeLbl">
         <property name="text">
          <string>Experiment Type:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="ExpTypeCmb">
         <property name="insertPolicy">
          <enum>QComboBox::InsertAfterCurrent</enum>
         </property>
         <item>
          <property name="text">
           <string>Eccentricity</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Polarangle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Horizontal meridians</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vertical meridians</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="SubjectNrLbl">
           <property name="text">
            <string>Subject No.:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSpinBox" name="subjectNrSpBx">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Subject Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="subjectNameEd"/>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="MRITab">
    <attribute name="title">
     <string>MRI</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="TRLbl">
         <property name="text">
          <string>TR:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="TRSpBx">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TRUnitLbl">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer1Tab2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="SimTriggerToggle">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Simulate Trigger</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QCheckBox" name="USBTriggerToggle">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>USB Trigger</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="USBCharLbl">
         <property name="text">
          <string>Character:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="USBCharEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="maxLength">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QCheckBox" name="COMTriggerOp">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>COM Trigger</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="COMAddLbl">
         <property name="text">
          <string>Register:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="COMSelectCmb">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>0x378</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0x278</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="expContentDkWdg">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>244</width>
     <height>271</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>191</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string> Experiment Content</string>
   </property>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QTableWidget" name="ExpContentTableWdg">
       <column>
        <property name="text">
         <string>Block/Trial</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Duration [TR]</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="SumOfTRsLbl">
         <property name="text">
          <string>Experiment duration:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NrOfTRsEd">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NrOfTRsUnitLbl">
         <property name="text">
          <string>TRs  </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QToolButton" name="AddBTBtn">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="RemoveBTBtn">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="SaveEXPFileBtn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>601</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="FullScreenToggle">
      <property name="text">
       <string>Full Screen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="DebugInfoToggle">
      <property name="text">
       <string>Info (test mode)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="GoBtn">
      <property name="text">
       <string>GO</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="blockTrialContentDkWdg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>625</width>
     <height>309</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>625</width>
     <height>296</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="blockTrialContentTbWdg">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QToolButton" name="AddTrialBtn">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="RemoveTrialBtn">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer100">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="SaveBlockTrialFileBtn">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="BlockTrialInfoGrpBx">
       <property name="title">
        <string>Info</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="conditionTypeLbl">
          <property name="text">
           <string>ConditionType:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="conditionTypeEd"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="nrOfParamsLbl">
          <property name="text">
           <string>No. of parameters:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="nameOfParametersLbl">
          <property name="text">
           <string>Names of parameters:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="typeOfParamsLbl">
          <property name="text">
           <string>Type of parameters:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="typeOfParamsEd"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="InfoTextLbl">
          <property name="text">
           <string>Info:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPlainTextEdit" name="InfoTxEd"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="nrOfParamsSpBx"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="nameOfParamsCmBx"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>USBTriggerToggle</sender>
   <signal>toggled(bool)</signal>
   <receiver>SimTriggerToggle</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>COMTriggerOp</sender>
   <signal>toggled(bool)</signal>
   <receiver>SimTriggerToggle</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>COMTriggerOp</sender>
   <signal>toggled(bool)</signal>
   <receiver>USBTriggerToggle</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>USBTriggerToggle</sender>
   <signal>toggled(bool)</signal>
   <receiver>COMTriggerOp</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>USBTriggerToggle</sender>
   <signal>toggled(bool)</signal>
   <receiver>COMSelectCmb</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>USBTriggerToggle</sender>
   <signal>toggled(bool)</signal>
   <receiver>COMAddLbl</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>COMTriggerOp</sender>
   <signal>toggled(bool)</signal>
   <receiver>USBCharLbl</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
